uber/site_sections/registration.py

```diff
@@ -401,7 +401,7 @@ def check_in(self, session, message='', group_id='', **params):
             attendee.checked_in = sa.localized_now()
             session.commit()
             increment = True
-            message += '{0.full_name} checked in as {0.badge}{0.accoutrements}'.format(attendee)
+            message += '{} checked in as {}{}'.format(attendee.full_name, attendee.badge, attendee.accoutrements)
```

Comment:
This format syntax _should_ have been fine, but I was getting errors for some strange reason!